ui/css/utilities/colors.scss

```diff
@@ -2,13 +2,8 @@
   // Accents
   // Everything below this line is part of the new color system
   --mainnet: #29b6af;
-  --goerli: #3099f2;
-  --sepolia: #cfb5f0;
-  --localhost: #bbc0c5;
-  --flask-purple: #8b45b6;
   --inherit: inherit;
   --transparent: transparent;
-
   // DO NOT CHANGE
   // Required for the QR reader to work properly
   --qr-code-white-background: #fff;
```

Comment:
Not used anywhere else in the code base

<img width="266" alt="Screen Shot 2022-11-16 at 1 12 06 PM" src="https://user-images.githubusercontent.com/8112138/202296436-21990d05-328a-4aec-a023-6221ea903214.png">
<img width="268" alt="Screen Shot 2022-11-16 at 1 12 11 PM" src="https://user-images.githubusercontent.com/8112138/202296439-725834fa-42be-41b2-a5e2-06b8c526cee9.png">
<img width="267" alt="Screen Shot 2022-11-16 at 1 12 15 PM" src="https://user-images.githubusercontent.com/8112138/202296442-82bfa3ae-a5c3-4ae6-b8de-36f458c731c8.png">
<img width="257" alt="Screen Shot 2022-11-16 at 1 12 20 PM" src="https://user-images.githubusercontent.com/8112138/202296444-bee1971e-1c4a-4ae9-bb8b-de9887b5b279.png">